src/main/groovy/com/bmuschko/gradle/docker/response/container/InspectContainerResponseHandler.groovy

```diff
@@ -20,10 +20,11 @@ class InspectContainerResponseHandler implements ResponseHandler<String, Object>
         logger.quiet "ExposedPorts : $exposedPorts"
         logger.quiet "LogConfig : $container.hostConfig.logConfig.type.type"
         logger.quiet "RestartPolicy : $container.hostConfig.restartPolicy"
-        def devices = container.hostConfig.@devices ? 
-                container.hostConfig.devices.collect { 
-                    "${it.pathOnHost}:${it.pathInContainer}:${it.cGroupPermissions}" 
+        def devices = container.hostConfig.@devices ?
+                container.hostConfig.devices.collect {
+                    "${it.pathOnHost}:${it.pathInContainer}:${it.cGroupPermissions}"
                 } : '[]'
         logger.quiet "Devices : $devices"
+        logger.quiet "ShmSize : $container.hostConfig.shmSize"
     }
```

Comment:
Can we remove the above changes? I'd like to just rip the ResponseHandler stuff out with the next release as we'll be going to 3.1.0.